Makefile

```diff
@@ -87,6 +87,9 @@ endif
 ldflags += $(LDFLAGS)
 ldflags := $(strip $(ldflags))
 
+build_tags += $(BUILD_TAGS)
+build_tags := $(strip $(build_tags))
```

Comment:
why changing the order?

Reply:
@robert-zaremba  so that the flags actually get in (instead of overwritten).

Would be great if this could get merged with 0.6.2

Reply:
the boltdb tags also get ignored in the old order...